.github/workflows/tools.yml

```diff
@@ -138,7 +138,7 @@ jobs:
       run: |
         pylint --rcfile ./styles/pylintrc ./tools/explorer
     - name: Ontology Explorer Tests
-      if: steps.changes.outputs.explorer == 'true'
+      if: steps.changes.outputs.explorer == 'true' OR steps.changes.outputs.type_validator == 'true' OR steps.changes.outputs.instance_validator == 'true'
```

Comment:
does the explorer relies on the instance validator? I dont think so

Reply:
`model_test.py` and `ontology_wrapper_test.py` rely on instance validator's simplified universe.